Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -448,7 +448,16 @@ public void Init(IServiceCollection serviceCollection)
 
             ConfigurationManager.AddParts(GetExports<IConfigurationFactory>());
 
-            NetManager = new NetworkManager(ConfigurationManager, _startupConfig, LoggerFactory.CreateLogger<NetworkManager>());
+            // this is needed as Github Codespaces will not be able to find the process if we bind on interfaces directly.
+
+            if (Environment.GetEnvironmentVariable("JELLYFIN_NETWORKMANAGER_BINDALL") == "true")
```

Comment:
```suggestion
            if (bool.TryParse(Environment.GetEnvironmentVariable("JELLYFIN_NETWORKMANAGER_BINDALL"), out var bindAll) && bindAll)
```

Doesn't really matter but might as well attempt to parse the bool

Reply:
![image](https://github.com/jellyfin/jellyfin/assets/6794763/10fd3a2f-1139-45ec-a72a-b94bb12b98fc)